src/platform/packages/shared/kbn-ebt-tools/src/performance_metrics/context/types.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+type ApmPageId = 'services' | 'traces' | 'dependencies';
+type InfraPageId = 'hosts';
+
+export type Key = `${ApmPageId}` | `${InfraPageId}`;
```

Comment:
Will all plugins have to define their page keys explicitly here? Is this to discourage changing prefixes?

Reply:
Yes, the idea is also to encourage consistency,  discoverability and avoid duplicate keys.